### ISSUE TITLE
Self-Hosted: Cannot use æøå in workflows

### ISSUE THREAD

**Issue by MarcusB13** [n8n-io/n8n#12098]
### Bug Description

I try to do an discord action when I receive an email from outlook. 
In the subject I receive æåø, and is not able to do anything with these letters. 

I cannot use them in code nodes nor in the discord node.

### To Reproduce

1. Create workflow
2. Receive data from any data source where æøå is included
3. Try to use this in a code node or anything else
4. When you try to use æøå in code node fx it will convert it to �

### Expected behavior

I expect the nodes and workflows to be able to save data like æøå

### Operating System

Ubuntu 22

### n8n Version

Newest. Master

### Node.js Version

v22.7.0

### Database

SQLite (default)

### Execution mode

main (default)

**Comment by Joffcom** [comment]
Hey @MarcusB13,

We have created an internal ticket to look into this which we will be tracking as "GHC-586"

**Comment by Joffcom** [comment]
@MarcusB13 It all depends on priority, At the moment this is waiting to be prioritised but it looks like an encoding issue. I have done a quick test and for me I see the characters as intended.

![image](https://github.com/user-attachments/assets/753acbfa-bfa9-4280-8f5e-7711c437c07e)

To do this I have used a form trigger node to get the data into n8n, Could it be that this is only an issue with emails? Are you using the IMAP trigger or the Outlook trigger node?

**Comment by MarcusB13** [comment]
Yes. I do get the same as you. But after saving it turns into a question mark symbol. Symbolising it's not recognized. This only happens after I have saved the workflow and reload
<img width="840" alt="Skærmbillede 2024-12-09 kl  15 05 09" src="https://github.com/user-attachments/assets/34ca9f80-4cc3-47ce-b86c-0f79bd652f7f">

**Comment by Joffcom** [comment]
Hey @MarcusB13,

I have tried to reproduce this on my local install and our cloud environment and for some reason on save and reload it still looks good. My initial thought was maybe it is down to the database but that seems to be ok.

Can you share more information on how you have n8n deployed? Do you have your browser set to a specific language or character set as well?

**Comment by Joffcom** [comment]
Hey @MarcusB13,

As we haven’t heard back from you, we’re going to close this issue to keep things organized. But don’t worry—if you have more information or need further assistance, you’re welcome to reopen the issue anytime or create a new one. We’re always happy to help! 😊

Thanks for reaching out and have a great day!
